packages/babel-helpers/src/helpers/applyDecs2203.js

```diff
@@ -0,0 +1,619 @@
+/* @minVersion 7.17.8 */
```

Comment:
Should it be `7.19.xx`?

Reply:
It needs to be updated when publishing, some tests fail when helpers specify a future version.